backend/src/routes/user.js

```diff
@@ -45,7 +45,13 @@ router.get('/', passport.authenticate('jwt', { session: false }), isValidBackend
 /**
  * Get profile information for a given user name
  */
-router.get('/:queryUserName', passport.authenticate('jwt', { session: false }), isValidBackendToken, getUserInfo)
+router.get('/:queryUserName', passport.authenticate('jwt', { session: false }), isValidBackendToken, (req, res) => {
+  if (req.query.providerProfile === 'false') {
```

Comment:
I feel like we should have separate endpoints for the two calls since they hit different CRM endpoints but I noticed that overall the user.js component does a mix of the two depending on what is required. So this should work for now.

Reply:
Left as is based on comments.